### DISCUSSION TITLE
I'm REALLLLY new to Docker and setting up plugins and such.

### DISCUSSION THREAD

**Discussion by qbjl** [itzg/docker-minecraft-server#923] [Q&A]
Hi! I read the docs and successfully ran a SPIGOT server. Now all I need help on is:

Getting onto the server (Currently for some reason it's hosting on 0.0.0.0)
Adding plugins (~Such as survival or decoration mods~ Edit: Finding the directory)
Running commands (/op)

And I think that's all. If there's a tutorial that answers my problems or something on the docs I didn't read, please tell me! I'm really new to hosting servers on Docker and wanted to host this server for survival :)

**Comment by itzg** [comment]
0.0.0.0 is just a special binding address that says listen on all interfaces. Just connect to your host's regular IP address. 

Plugins: https://github.com/itzg/docker-minecraft-server#working-with-mods-and-plugins

Running commands: https://github.com/itzg/docker-minecraft-server#interacting-with-the-server
